output/json/json.go

```diff
@@ -171,6 +170,29 @@ func (o *Output) handleMetric(m *metrics.Metric, jw *jwriter.Writer) {
 	}
 	o.seenMetrics[m.Name] = struct{}{}
 
-	wrapMetric(m).MarshalEasyJSON(jw)
+	//nolint:exhaustruct
```

Comment:
I think this is a linter added after the version we currently use so I would prefer if you don't add `nolint` for it but maybe downgrade your local version (I just did that yesterday). 

Preferably this should be fixed in the tool ... but I guess we will just update the version we use ... soon :tm: